gnd/src/main/java/com/google/android/gnd/persistence/remote/FirestoreStorageManager.java

```diff
@@ -39,20 +35,18 @@
 
   private static final String TAG = FirestoreStorageManager.class.getName();
   private static final String MEDIA_ROOT_DIR = "uploaded_media";
-  private final SimpleDateFormat dateFormat =
-      new SimpleDateFormat("yyyyMMddHHmmss", Locale.getDefault());
 
   @Inject
   FirestoreStorageManager() {}
 
   /** Returns a reference to the default Storage bucket. */
-  private FirebaseStorage getStorage() {
-    return FirebaseStorage.getInstance();
+  private StorageReference getBaseReference() {
+    return FirebaseStorage.getInstance().getReference();
```

Comment:
Can we instead inject FirebaseStorage or StorageReference to make it mockable in tests?